src/PendingReason.php

```diff
@@ -235,13 +256,51 @@ public static function displayFollowupsNumberBeforeResolutionField(
         return Dropdown::showFromArray($name, $values, $options);
     }
 
+    /**
+     * Display specific "is_default" field
+     *
+     * @param $value
+     * @param $name
+     * @param $options
+     */
+    public function displayIsDefaultPendingReasonField(
+        $value = null,
+        $name = "",
+        $options = [],
+    ) {
```

Comment:
1. Please use strict types when possible.
2. Some parameters are never passed.
3. If method is only used in private scope, make it private.
```suggestion
    private function displayIsDefaultPendingReasonField(bool $value): string {
```